frontend/app/src/components/asset-manager/managed/ManagedAssetIgnoringMore.vue

```diff
@@ -29,26 +40,32 @@ const toggleWhitelist = () => {
       transition="slide-y-transition"
     >
       <template #activator="{ on }">
-        <RuiButton class="!p-2" icon variant="text" v-on="on">
-          <RuiIcon name="more-2-fill" size="20" />
+        <RuiButton class="!p-1" icon variant="text" v-on="on">
+          <RuiIcon name="arrow-down-s-line" size="20" />
         </RuiButton>
       </template>
       <div class="py-2 text-rui-text-secondary">
-        <RuiButton
-          v-if="!isWhitelisted"
-          variant="list"
-          @click="toggleWhitelist()"
-        >
+        <RuiButton variant="list" @click="toggleWhitelist()">
           <template #prepend>
-            <RuiIcon name="checkbox-circle-line" />
+            <RuiCheckbox
+              color="primary"
+              class="[&_span]:-mr-1 [&_span]:!py-0"
```

Comment:
Isn't this something that we should do as part of the component itself instead of overriding the css outside?